accelerator/errors.h

```diff
@@ -79,6 +83,10 @@ typedef enum {
   /**< JSON key not found */
   SC_CCLIENT_JSON_PARSE = 0x08 | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
   /**< json parsing error, might the wrong format */
+  SC_CCLIENT_FLEX_TRITS = 0x09 | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
+  /**< Flex trits converting error */
+  SC_CCLIENT_JSON_CREATE = 0x0A | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
```

Comment:
@wusyong we should expand our status code bits, error code is 8 at most.
Maybe open another cleanup issue to modify?

Reply:
Your are right. We shall update to mapping.